docs/manuals/source/DeveloperGuide/api.rst

```diff
@@ -183,20 +183,20 @@ API mode 1 (on).
 
     .api 1
 
-or (form times where they have only been one API flavour)
+or (from times where they was only one API flavour)
 
 ::
 
     .api
 
-This mode is intended to create output that is earlier parsable.
-Internaly some commands vary there output for the API mode 1, but not
+This mode is intended to create output that is earlier to parse.
+Internaly some commands vary their output for API mode 1, but not
 all.
 
-In API mode 1 some output is only delimted by the end of a packet, by
-not a new line. bconsole does not display end of packets (for good
-reason, as some output (e.g. :bcommand:`status`) is send in multiple packets).
-If running in a bconsole, this leads not parsable output for human.
+In API mode 1 some output is only delimited by the end of a packet, not
+by a new line. bconsole does not display the end of packets (for good
+reason, as some output (e.g. :bcommand:`status`) is sent in multiple packets).
+If running in a bconsole, this leads to unparsable output for human.
```

Comment:
```suggestion
In API mode 1 some output is delimited by end of a packet and not by a newline character.
Thus the output of API mode 1 may be non-parseable when being used in bconsole, as end of packet is not displayed there.
```

Reply:
If I remember correctly, the output is displayed. It is just very hard to read, as line-breaks are missing.